spec/std/enum_spec.cr

```diff
@@ -88,6 +102,24 @@ describe Enum do
     end
   end
 
+  describe "size" do
+    it "for simple enum" do
+      SpecEnum.size.should eq 3
+    end
+
+    it "for flags enum" do
+      SpecEnumFlags.size.should eq 3
+    end
+
+    it "gives number of unique enum values" do
+      SpecNonUniqueEnum.size.should eq 2
```

Comment:
IMO this should count number of enum *members*, not values.

Reply:
That's the big question of this PR I guess, but I can't find a usecase for this behavior..
Do you have one?

If the 2 behaviors are legit, maybe we can have 2 functions `size` & `uniq_size` or sth..

Reply:
There's no need for uniqness if we're talking about _members_.

Reply:
For me `size` should be the number of members. If you want to know the number of uniq values, select all members and call `uniq`.